tests/Aspire.Hosting.Tests/Utils/LoggerNotificationExtensions.cs

```diff
@@ -62,7 +82,7 @@ private static async Task WatchNotifications(DistributedApplication app, string?
         {
             await foreach (var resourceEvent in resourceNotificationService.WatchAsync(cancellationTokenSource.Token).ConfigureAwait(false))
             {
-                if (resourceName != null && !string.Equals(resourceEvent.Resource.Name, resourceName, StringComparisons.ResourceName))
+                if (resourceName != null && !string.Equals(resourceEvent.Resource.Name, resourceName, StringComparison.OrdinalIgnoreCase))
```

Comment:
Isn't it the same?
` public static StringComparison ResourceName => StringComparison.OrdinalIgnoreCase;`

Reply:
Because of the include, the `StringComparisons` is not accessible.